packages/forms/src/validators.ts

```diff
@@ -499,6 +499,10 @@ export function maxLengthValidator(maxLength: number): ValidatorFn {
   };
 }
 
+export function toInteger(digit: string, parseType?: string): number {
+  return parseType === 'float' ? parseFloat(digit) : parseInt(digit, 10);
+}
```

Comment:
I'd propose keeping the old logic, but just moving it to a different function:

```suggestion
export function toInteger(value: string|number): number {
  return typeof value === 'number' ? value : parseInt(value, 10);
}
```

The problem with the current approach is that we'd convert number -> string before passing into a function and then convert it back to number, which is all not needed and would be just faster to use `typeof` check. Regarding `parseFloat` support - I'd not add it right now and update the code as needed once min/max validators are updated.